src/redesign/_style.scss

```diff
@@ -667,3 +667,7 @@ select.form-control {
   max-height: 1rem;
   margin-top:6px;
 }
+
+.pgn__icon {
```

Comment:
That would change the width of all the icons. Make it specific to that icon.

Reply:
changes addressed!